src/app/components/common/GlossyButton.jsx

```diff
@@ -0,0 +1,28 @@
+"use client";
```

Comment:
do you need use client here?

Reply:
![Screen Shot 2024-06-26 at 12 08 13 PM](https://github.com/Jula000/photostudio/assets/128170039/99dc473e-68d7-46c6-9a3b-83938c0ac1ef)